src/View/Fight/index.html.twig

```diff
@@ -0,0 +1,47 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}Combat
+{% endblock %}
+
+{% block stylesheet %}
+  {{ parent() }}
+  <link rel="stylesheet" type="text/css" href="/assets/css/fight.css"/>
+  <link rel="stylesheet" type="text/css" href="/assets/css/style.css">
```

Comment:
you don't need style.css here since this global file is automatically injected inside layout file